docs/arcadia-users-group/20221017-conda/lesson.md

```diff
@@ -0,0 +1,369 @@
+
+## What is Conda?
+
+Conda is open source package and runs on Windows, Mac OS and Linux.
+
++ Conda can quickly install, run, and update packages and their dependencies.
++ Conda can create, save, load, and switch between project specific software environments on your local computer.
+
+Conda as a package manager helps you find and install packages. 
+If you need a package that requires a different version of Python, you do not need to switch to a different environment manager, because Conda is also an environment manager. 
+With a few commands, you can set up a totally separate environment to run that different version of Python, while continuing to run a different version of Python in another environment.
+
+## How does Conda work?
+
+<center>
+<figure markdown>
+  ![conda](conda.png){ width="700" }
+  <figcaption> Cartoon of conda environments by Gergely Szerovay <a href='https://www.freecodecamp.org/news/why-you-need-python-environments-and-how-to-manage-them-with-conda-85f155f4353c/' target='_blank'>www.freecodecamp.org</a> </figcaption>
+</figure>
+</center>
+
+### Installing Conda
+
+<center>
+<figure markdown>
+  ![installingconda](installing_conda.png){ width="700" }
+  <figcaption> Cartoon of decision points for conda installation by Gergely Szerovay <a href='https://www.freecodecamp.org/news/why-you-need-python-environments-and-how-to-manage-them-with-conda-85f155f4353c/' target='_blank'>www.freecodecamp.org</a> </figcaption>
+</figure>
+</center>
+
+For this lesson, we'll install miniconda. 
+We've included the latest installation instructions below.
+You can access the latest and legacy installation links [here](https://docs.conda.io/en/latest/miniconda.html#latest-miniconda-installer-links).
+
+#### Installing conda on an Apple machine
+
+Apple now has two processor types, the Intel x64 and the Apple M1 (or M2) ARM64.
+As of October 2022, many of the packages available via conda-forge and other conda installation channels are only available of the Intel x64 processor. 
+Therefore, even if you have an M1 (or M2) Mac, we currently recommend that you use the Intel x64 installation and take advantage of Apple's translation program Rosetta. 
+This requires a little bit of pre-configuration to make sure your Terminal application also runs with Rosetta.
+To set this up, open **Finder** -> navigate to your **Applications** folder -> right click on your Terminal application (either Terminal or iTerm2) -> select **Get Info** -> check the box for **Open using Rosetta**.
+
+<center>
+<figure markdown>
+  ![installingconda](rosetta.png){ width="200" }
+  <figcaption> Running Terminal/iTerm using Rosetta </figcaption>
+</figure>
+</center>
+
+After that is configured, open your Terminal application and copy and paste the following commands to download and install miniconda.
+
+```
+curl -JLO https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
```

Comment:
I'd add a note on versioning. And how to install specific versions if you need it for consistency since the conda APIs may change between versions etc.